vars/artifactSetVersion.groovy

```diff
@@ -32,7 +32,7 @@ def call(Map parameters = [:], Closure body = null) {
         def gitUtils = parameters.juStabGitUtils ?: new GitUtils()
 
         if (gitUtils.insideWorkTree()) {
-            if (sh(returnStatus: true, script: 'git diff --quiet HEAD') != 0)
+            if (gitUtils.isWorkTreeDirty())
```

Comment:
As stated before, personally I don't like the duplicate check for `insideWorkTree`, but feel free to ignore my voting.